src/Administration/Resources/app/administration/test/module/sw-product/component/sw-product-media-form.spec.js

```diff
@@ -129,4 +137,14 @@ describe('module/sw-product/component/sw-product-media-form', () => {
         const pageChangeEvents = wrapper.emitted()['media-open'];
         expect(pageChangeEvents.length).toBe(1);
     });
+
+    it('should remove the cover properly if all medias removed', async () => {
+        const wrapper = createWrapper();
+
```

Comment:
Can you also add a check if a product cover was set before :)

Reply:
Sorry I think I do not quite understand, do you mean to verify that `createWrapper` properly creates a `cover`?

Reply:
Should be fixed.